docs/data_model.md

```diff
@@ -197,94 +195,16 @@ that the parent domain is an authority for the service itself, from a management
 perspective. 
 
 A Service Identity may be used as a principal when specifying Roles, just like a
-User. Athenz provides support for registering such a Service Identity, in a domain,
-along with its public key that can be used to later verify an NToken that is
-presented by the service.
-
-## Tokens
---------
-
-Athenz authorization system utilizes 2 types of tokens: Principal Tokens
-(NTokens) and RoleTokens (ZTokens).
-
-### Principal Token (NToken)
```

Comment:
Let's not remove the Principal Token section - this is still valid and might be used by properties. Only role token references should be removed.